grails-app/conf/Config.groovy

```diff
@@ -128,7 +128,7 @@ openId {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    filterSource = 'database' // Todo - DN: This can go back to being a boolean when we've removed loading filters from the DB
```

Comment:
I think leave it as is, even though it's not a toggle. no_db will clean this up, hopefully it's just "around the corner".